<ui version="4.0" >
 <class>EnginePrimitivesWidget</class>
 <widget class="QWidget" name="EnginePrimitivesWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>335</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="Avogadro::PrimitiveTreeView" name="enginePrimitivesTree" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="animated" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="buttonFrame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QToolButton" name="addSelectionButton" >
        <property name="toolTip" >
         <string>Add Selected Primitives</string>
        </property>
        <property name="text" >
         <string>Add Selected Primitives</string>
        </property>
        <property name="icon" >
         <iconset resource="avogadro.qrc" >
          <normaloff>:/icons/edit-add.png</normaloff>:/icons/edit-add.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="removeSelectionButton" >
        <property name="toolTip" >
         <string>Remove Selected Primitives</string>
        </property>
        <property name="text" >
         <string>Remove Selected Primitives</string>
        </property>
        <property name="icon" >
         <iconset resource="avogadro.qrc" >
          <normaloff>:/icons/edit-remove.png</normaloff>:/icons/edit-remove.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="resetButton" >
        <property name="toolTip" >
         <string>Add All Primitives</string>
        </property>
        <property name="text" >
         <string>Add All Primitives</string>
        </property>
        <property name="icon" >
         <iconset resource="avogadro.qrc" >
          <normaloff>:/icons/edit-undo.png</normaloff>:/icons/edit-undo.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="fromSelectionButton" >
        <property name="toolTip" >
         <string>Display Only Selected Primitives</string>
        </property>
        <property name="text" >
         <string>Assign to Selection</string>
        </property>
        <property name="icon" >
         <iconset resource="avogadro.qrc" >
          <normaloff>:/icons/selecttable.png</normaloff>:/icons/selecttable.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Avogadro::PrimitiveTreeView</class>
   <extends>QTreeView</extends>
   <header>primitivetreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="avogadro.qrc" />
 </resources>
 <connections/>
</ui>
